src/validator/CreditCardVendor.java

```diff
@@ -0,0 +1,32 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardVendor {
+	private String name;
+	private ArrayList<Integer> numberOfChars;
```

Comment:
Consider changing ArrayList to Set code will be simpler when validating card number